Formula/d/dynare.rb

```diff
@@ -48,6 +49,11 @@ class Dynare < Formula
     sha256 "fa80f7c75dab6bfaca93c3b374c774fd87876f34fba969af9133eeaea5f39a3d"
   end
 
+  patch do
```

Comment:
Can this get a comment explaining what it's doing/why it's needed/upstream bug report?